cartoframes/assets/src/widgets.js

```diff
@@ -2,18 +2,26 @@ import { format } from './utils';
 
 export function renderWidget(widget, value) {
   widget.element = widget.element || document.querySelector(`#${widget.id}-value`);
-  
+
   if (value && widget.element) {
     widget.element.innerText = typeof value === 'number' ? format(value) : value;
   }
 }
 
-export function renderBridge(bridge, widget) {
+export function renderBridge(bridge, widget, mapLayer) {
   widget.element = widget.element || document.querySelector(`#${widget.id}`);
+  const type = mapLayer.metadata.properties[widget.value].type;
+
+  console.log('!!! renderBridge', widget);
 
   switch (widget.type) {
     case 'histogram':
-      bridge.histogram(widget.element, widget.value, widget.options);
+      if (type === 'category') {
```

Comment:
This is way better. I really don't like the implicit 'histogram' that decides for you, and it was my idea  sweat_smile

Reply:
I know 👌 , I'd like to change the `buckets` check in Airship at some point, but right now this is great to make it work in CARTOframes without releasing a new version for Airship.

Reply:
I'd leave that bridge.histogram for compatibility's sake, but I would never use it personally :man_shrugging: